internal/impl/devbox.go

```diff
@@ -735,7 +737,11 @@ func (d *Devbox) computeNixEnv(ctx context.Context, usePrintDevEnvCache bool) (m
 		if ignoreCurrentEnvVar[key] {
 			continue
 		}
-		env[key] = val
+		// Need HOME env variable for pure shell to leak through
+		// otherwise devbox binary won't work
+		if !pure || key == "HOME" {
```

Comment:
Should this be `HOME, USER and DISPLAY` to match nix?

Reply:
yeah sure, I want to keep it as minimal as possible. But matching nix is ok in this case.